conversation_classification/kaggle/model.py

```diff
@@ -28,9 +28,15 @@
 MAX_DOCUMENT_LENGTH = 500 # Max length of each comment in words
 
 # Model Params
```

Comment:
These lists are getting a bit hard to distinguish. I'd recommend making two dictionaries of default parameters: CNN_DEFAULT_PARAMS and BOW_DEFAULT_PARAMS. You can later add the functionality to update these dictionaries with command line flags.

Reply:
Great idea.